src/misc/fs_utils_win32.cpp

```diff
@@ -95,7 +95,7 @@ NativeFileHandle open_native_file(const std_fs::path& path, const bool write_acc
 
 	return CreateFileW(path.c_str(),
 	                   access,
-	                   FILE_SHARE_READ | FILE_SHARE_WRITE | FILE_SHARE_DELETE,
+	                   FILE_SHARE_READ | FILE_SHARE_WRITE,
```

Comment:
Would be a perfectly acceptable and clean fix if it worked reliably. Great find @weirddan455, let's see how the testing goes 👍🏻